todo_app/tests/test_client.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pytest
+import requests
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
+
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+def test_index_page(client):
+
+    response = client.get('/')
+
+def test_index_page(monkeypatch, client):
+    # This replaces any call to requests.get with our own function
+    monkeypatch.setattr(requests, 'get', stub)
+#    monkeypatch.setattr(requests, 'request', stub)
+
+    response = client.get('/')
```

Comment:
In fact, we have the same thing here - this gets replaced in turn by the version of the test defined at the bottom!
```suggestion
```